analyses/cell-type-consensus/scripts/00-download-panglao-ref.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script is used to download the reference file for PanglaoDB from scpca-nf
+# The downloaded file will be saved to `references/panglao-cell-type-ontologies.tsv`
+
+set -euo pipefail
+
+# navigate to where script lives
+cd $(dirname "$0")
+scripts_dir=$(pwd)
+
+# define path to ref file and url
+ref_file="${scripts_dir}/../references/PanglaoDB_markers_2020-03-27.tsv"
+ref_url="https://raw.githubusercontent.com/AlexsLemonade/scpca-nf/refs/heads/main/references/PanglaoDB_markers_2020-03-27.tsv"
```

Comment:
I suppose `main` is fine here (instead of a permalink) since the file name captures version information, and we'd probably want to use whatever the current version of this file is anyway.

Reply:
> and we'd probably want to use whatever the current version of this file is anyway.

This exact thought was my reasoning for using `main` here.